src/components/Reports/GDPRPermissions/PermissionExpandableRow.test.js

```diff
@@ -0,0 +1,184 @@
+    permissionHTML:
+      '<p>Acepta las promociones indicadas <a href="http://www.google.com">aqui</a></p>',
+  };
+
+  const field_value_none = {
+    name: 'Permiso2',
+    value: 'none',
+    predefined: false,
+    private: false,
+    readonly: false,
+    type: 'permission',
+    permissionHTML:
+      '<p>Acepta las promociones indicadas <a href="http://www.google.com">aqui</a></p>',
+  };
+
+  const dopplerApiClientDouble = {
+    getUserFields: async () => {
+      return { success: true, value: subscriber.fields };
+    },
+  };
+
+  const experimentalFeatureDouble = () => {
+    return {
+      getFeature: (feature) => feature === 'PermissionHistory',
+    };
+  };
+
+  const dependencies = {
+    dopplerApiClient: dopplerApiClientDouble,
+    experimentalFeatures: experimentalFeatureDouble(),
+  };
+
+  afterEach(cleanup);
+
+  it('renders PermissionExpandableRow without error', async () => {
+    // Arrange
+    // Act
+    render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <IntlProvider>
+          <table>
+            <tbody>
+              <PermissionExpandableRow field={field} />
+            </tbody>
+          </table>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+  });
+
+  it('should show the arrow button if the value of the field is not none', () => {
+    // Arrange
+    // Act
+    const { container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <IntlProvider>
+          <table>
+            <tbody>
+              <PermissionExpandableRow field={field} />
+            </tbody>
+          </table>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.dp-expand-results')).toBeInTheDocument();
+  });
+
+  it("shouldn't show the arrow button if the value of the field is none", () => {
+    // Arrange
+    // Act
+    const { container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <IntlProvider>
+          <table>
+            <tbody>
+              <PermissionExpandableRow field={field_value_none} />
+            </tbody>
+          </table>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.dp-expand-results')).not.toBeInTheDocument();
+  });
+
+  it("shouldn't have the class 'show' if the arrow button is not clicked at least once", () => {
+    // Arrange
+    // Act
+    const { container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
+        <IntlProvider>
+          <table>
+            <tbody>
+              <PermissionExpandableRow field={field} />
+            </tbody>
+          </table>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    expect(container.querySelector('.dp-expanded-table').classList.contains('show')).toBe(false);
+  });
+
+  it("should have the class 'show' when the arrow button is clicked", () => {
```

Comment:
What do you think about testing if the content is expanded in place of checking the class?

```suggestion
  it("should be expanded when the arrow button is clicked", () => {
```

And then, in the assert verify the content in place of a class:

```diff
-     expect(container.querySelector('.dp-expanded-table').classList.contains('show')).toBe(true);
+     // Pseudo code: verify if the inner table is rendered with the right data.
```